src/app/exam/ExamProblems.scss

```diff
@@ -0,0 +1,27 @@
+
+.container {
+  padding: 40px 15rem 20px 15rem;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.add-problem-btn {
+  background-color: #7ECA1D;
+  box-shadow: 1px 3px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 50px;
+  float: right;
+  display: inline;
+  margin: 1rem 0;
+  padding: 5px 20px;
+  cursor: pointer;
+
+  span {
+    color: #FFF;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 30px;
+
+  }
+}
```

Comment:
Namespacing: prefix all styles with the component name 

example: https://github.com/Judge-Girl/Judge-Girl-CMS/blob/master/src/app/exam/modals/AddExamineesModal.scss

Reply:
@edisonhello still not resolved